docker/Dockerfile.alpine

```diff
@@ -1,40 +1,26 @@
-FROM alpine:3.8
+FROM alpine:3.11
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.10/main' >> /etc/apk/repositories
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/edge/community' >> /etc/apk/repositories
 RUN apk add --update \
   bison \
-  binutils-dev \
+  bcc-dev==0.12.0-r2 \
```

Comment:
This package revision hasn't made it into 3.11 yet (waiting on reviewers), so I'm fetching these from edge.

Reply:
I don't think even the dev package has static libraries we need, as the PR for this was literally just merged

Reply:
I added a bcc-static subpackage for this purpose -- see the next line -- and it's available in edge. The second revision landed yesterday and includes libbccbpf.a and bcc's static loader. Package contents [here](https://pkgs.alpinelinux.org/contents?branch=edge&name=bcc-static&arch=x86_64&repo=community).

Reply:
ah good, that'll save ~5 minutes of CI time.

Reply:
@acj I suggest you template the version number and expose it as a build arg, like in travis.yml

By convention the other dockerfiles do this for LLVM version, and I was doing it for BCC version in #1041